### DISCUSSION TITLE
Fill in the grid

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#958] [Galaxy Trivia] [answered]
```
Y G W A
K L ? O
Z J P D
V F T H
```

**Comment by peterbaccaro** [comment, marked as answer]
The missing letter is **R** because the total of the alphabet positions of each letter in a row is always 56.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/86f3f8a8-3634-4b78-b3a8-03e8d968d8d7)

**Comment by thuongtruong1009** [comment]
Based on the given grid, here is the completed version:

Y G W A
K L X O
Z J P D
V F T H

In the second row, the missing letter is "X" to complete the sequence.
